### DISCUSSION TITLE
Save colors in pane:get_lines_as_text?

### DISCUSSION THREAD

**Discussion by stevalkr** [wezterm/wezterm#6365] [General]
Hi! I’m trying to use neovim to open the scrollback followed by [this](https://wezfurlong.org/wezterm/config/lua/pane/get_lines_as_text.html?h=scrollback+vim#example-opening-whole-scrollback-in-vim), but keep the colored text. ([kitty-scrollback.nvim](https://github.com/mikesmithgh/kitty-scrollback.nvim)) supports that. How can I do the same?

**Comment by bew** [comment]
Have you tried [`pane:get_lines_as_escapes(...)`](https://wezfurlong.org/wezterm/config/lua/pane/get_lines_as_escapes.html) which includes all escapes including the ones for color/style/.. ?

**Comment by stevalkr** [comment]
Thank you! This solves my problem
